docs/tutorials/debug_tools_for_tilelang.md

```
@@ -194,8 +194,262 @@ C_local inferenced layout:
+### Environment Variables
+
+| Variable | Description | Default |
+|----------|-------------|---------|
+| `TL_LOWER_TRACE` | Enable tracing. Values: `0`/`off`/`false`/`no` (off), `1`/`on`/`true`/`yes` (→ html), `terminal`, `html`, `both` | off |
+| `TL_LOWER_TRACE_DIR` | Base output directory for all trace artifacts | `./tmp/lower_trace_dir` |
+
+### Output Directory Structure
+
+A single run produces the following layout under `TL_LOWER_TRACE_DIR`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)